### DISCUSSION TITLE
Does gephi have a command line tool?

### DISCUSSION THREAD

**Discussion by wangbinkarl** [gephi/gephi#2792] [Q&A]
Does gephi have a command line tool?

**Comment by mbastian** [comment]
Hi @wangbinkarl, what exactly are you trying to achieve. Gephi doesn't have a command line tool but if you're looking to automate what Gephi does but programatically, the https://github.com/gephi/gephi-toolkit is a library that you can use in your own program. Creating a command-line tool based on the Toolkit should be pretty easy.
